README.md

```diff
@@ -24,7 +24,7 @@ grunt.loadNpmTasks('grunt-contrib-qunit');
 ## Qunit task
 _Run this task with the `grunt qunit` command._
 
-You have chosen to write your unit tests using [QUnit][], you have written a
+You have chosen to write your unit tests using [QUnit](http://qunitjs.com/), you have written a
```

Comment:
@leobalter & @Arkni Thank you for your assistance.  The second push attempt prompted for credentials then succeeded.